.pre-commit-config.yaml

```
@@ -46,3 +46,9 @@ repos:
         entry: python docs/source/parameters_doc.py
         language: system
         files: ^docs/source/parameters_doc\.py$
+    -   id: generate-keras-ops
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)